src/App.jsx

```diff
@@ -1,11 +1,7 @@
-import Index from "./components/One/index";
+import Hero from "./components/HeroSection/Hero";
+
 function App() {
-  return (
-    <div className="flex flex-col items-center justify-center font-bold p-80">
-      <p className="text-pink-800">Project 2</p>
-      <Index />
-    </div>
-  );
+  return <Hero />;
```

Comment:
put your component inside a `<div></div>` since everyone is going to put there components here only